feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -335,7 +363,7 @@ func contains(item string, items []string) bool {
 	return flag
 }
 
-// verifyTraffic confirms that every traffic flow has the expected amount of loss (0% or 100%
+// verifyTraffic confirms that every traffic flow has the expected amount of loss (0% or 100%).
 func verifyTraffic(t *testing.T, ate *ondatra.ATEDevice, flows []*ondatra.Flow, passFlows []string) {
```

Comment:
This test will be more type safe if we change passFlows to be of type []*ondatra.Flow. The contains check below then just be "if contains(flow, passFlows) {". And then the passFlows provided by the tcs struct can be the actual flow references instead of untyped strings.

Reply:
done in new commit.